src/components/interface/SearchBar/SearchBar.js

```diff
@@ -16,7 +16,6 @@ import '@gouvfr/dsfr/dist/css/inputs.min.css';
 /*
 * react-ds-fr
 */
-import '../../../style/colors.css';
```

Comment:
Where do we keep `colors.css` import finally?

Reply:
it's managed by the dsfr directly